hmy/staking.go

```diff
@@ -215,9 +215,23 @@ func (hmy *Harmony) getEpochSigning(epoch *big.Int, addr common.Address) (stakin
 	}
 
 	// the signing information is for the previous epoch
-	entry.Epoch = big.NewInt(0).Sub(epoch, common.Big1)
+	prevEpoch := big.NewInt(0).Sub(epoch, common.Big1)
+	entry.Epoch = prevEpoch
 	entry.Blocks = snapshot.Validator.Counters
 
+	// subtract previous epoch counters if exists
+	prevEpochSnap, err := hmy.BlockChain.ReadValidatorSnapshotAtEpoch(prevEpoch, addr)
```

Comment:
if err return error too.

Reply:
we don't want to return error if the previous snapshot does not exists, it is a valid case.